gcloud-java-dns/src/main/java/com/google/gcloud/dns/DnsImpl.java

```diff
@@ -303,7 +304,7 @@ public ChangeRequest getChangeRequest(final String zoneName, final String change
                   return dnsRpc.getChangeRequest(zoneName, changeRequestId, optionsMap);
                 }
               }, options().retryParams(), EXCEPTION_HANDLER);
-      return answer == null ? null : fromPb(answer);
+      return answer == null ? null : ChangeRequest.fromPb(this, zoneName, answer);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'ChangeRequest.fromPb' due to existing static import 'com.google.gcloud.dns.ChangeRequest.fromPb'](https://www.codacy.com/app/mziccard/gcloud-java/file/2311199859/issues/source?bid=3123106&fileBranchId=3198278#l307)